packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -290,15 +285,15 @@ export class HttpPlugin extends classes.BasePlugin {
         response.on('end', () => {
           plugin.logger.debug('outgoingRequest on end()');
           const method = response.method ? response.method : 'GET';
-          const reqUrl = url.parse(options);
           const userAgent =
-              headers ? (headers['user-agent'] || headers['User-Agent']) : null;
+              headers ? (headers['user-agent'] || headers['User-Agent']) : '';
```

Comment:
If there is no user agent... will it append an empty label? We should make the label conditional on there being a user agent.

Reply:
Done!